src/App.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from "react";
-
 import { MarkerFileData } from "./DataTypes";
 import RenderView from "./RenderView";
+import { useFilePicker } from 'use-file-picker';
```

Comment:
You'll also have to commit `package.json` and `package-lock.json` with the new `use-file-picker` dependency